src/Tickets/Commerce/Hooks.php

```diff
@@ -93,6 +93,10 @@ protected function add_filters() {
 
 		// Add a post display state for special Event Tickets pages.
 		add_filter( 'display_post_states', [ $this, 'add_display_post_states' ], 10, 2 );
+
+		// @todo make the following filters dynamic.
```

Comment:
could we add a person responsible for this + a jira ticket?

Reply:
Actually can we make it into a dynamic thing now? It shouldn't be hard.